### PR TITLE
Containers: skip tests that are not supported on certain versions

### PR DESCRIPTION
**Pull request by jlausuch**
VRs:
- 12-SP3: [before](https://openqa.suse.de/tests/6292056) -> [after](https://openqa.suse.de/tests/6294177)
- 12-SP4: [before](https://openqa.suse.de/tests/6292157) -> [after](https://openqa.suse.de/tests/6294175)
- 12-SP5: [before](https://openqa.suse.de/tests/6292247) -> [after](https://openqa.suse.de/tests/6294174)
- 15: [before](https://openqa.suse.de/tests/6292408) -> [after](https://openqa.suse.de/tests/6294171)

### CODE REVIEW

File: schedule/containers/engines_and_tools.yaml [hunk 1]
```diff
@@ -6,24 +6,45 @@ description:    >
...
     ARCH:
       x86_64:
         - containers/validate_btrfs
+  runc:
+    ARCH:
+      x86_64:
+        - containers/docker_runc
+      s390x:
+        - containers/docker_runc
+  supported:
```
Comment (b10n1k) [inline schedule/containers/engines_and_tools.yaml:21]: why not something that mentions podman (ex: podman_supported) or just podman?

Comment (jlausuch) [reply schedule/containers/engines_and_tools.yaml:21]: cause it's also for buildah package, `buildah_podman`?

**Review by pdostal — APPROVED** [review]

**Comment by pdostal** [comment]
If no other objections I will merge at the evening.

File: schedule/containers/engines_and_tools.yaml (hunk 1 above)
Comment (b10n1k) [reply schedule/containers/engines_and_tools.yaml:21]: i guess we can leave it as it is. We can always change it in another time

**Pull request merged**